tests/Doctrine/Migrations/Tests/Version/ExecutorTest.php

```diff
@@ -245,6 +246,18 @@ protected function setUp() : void
         $stopwatchEvent->expects(self::any())
             ->method('getMemory')
             ->willReturn(100);
+
+        $stopWatchPeriod = $this->createMock(StopwatchPeriod::class);
+        $stopWatchPeriod->expects(self::any())
```

Comment:
```suggestion
        $stopWatchPeriod
```

Reply:
I quite agree with this change, but why changing it here only, and not in all other places? Like the existing `$stopwatchEvent->expects(self::any())`?

Reply:
Feel absolutely free to :), but in a separate commit please.

Reply:
Sorry, I amended my previous commit :(